index.js

```diff
@@ -7,6 +7,15 @@ const { ReconciliationText } = require("./lib/reconciliationText");
 const { SharedPart } = require("./lib/sharedPart");
 const { firmCredentials } = require("./lib/api/firmCredentials");
 
+async function fetchReconciliation(firmId, handle) {
+  const templateConfig = fsUtils.readConfig("reconciliationText", handle);
+  if (templateConfig || templateConfig.id[firmId]) {
```

Comment:
I think we should do an "&&" check instead of "||"? 

Cause templateConfig could be true if the config.json was already created from importing from another firm and so this if-statement would pass, but the ID from the specific firm you want to import from doesn't exist yet.

Reply:
Yes, you are right! :see_no_evil: